app/models/fae/navigation.rb

```diff
@@ -69,9 +72,28 @@ def item(text, options={})
       # prefer `subitems` over `sublinks` for new DSL but still need to support old structure in v1
       hash[:sublinks] = hash[:subitems] if hash[:subitems].present?
 
+      # authorize link with current user
+      unless can_view_path(hash[:nested_path])
+        hash[:path] = '#'
```

Comment:
Ah, I see. My bad for not seeing the flow of everything. LDT, but I think it'd be worth noting in the `navigation_concern.rb` docs that any `href="#"` without sublinks will be removed from nav items, because that isn't expected behavior. And I can see some case arising where the nav hosts some custom JS functionality using faux `href="#"` links.